linalg/petsc.cpp

```diff
@@ -834,7 +834,7 @@ void PetscParMatrix::ConvertOperator(MPI_Comm comm, const Operator &op, Mat* A,
                if (tid == PETSC_MATIS && needl2l)
                {
                   PetscContainer c;
-                  ierr = PetscObjectQuery((PetscObject)mats[i*nc+j],"__mfem_l2l",
+                  ierr = PetscObjectQuery((PetscObject)mats[i*nc+j],"_MatIS_PtAP_l2l",
```

Comment:
I'm not sure what the role of the modified string is: does this change mean that MFEM now requires PETSc 3.9.0 or newer?